apps/dotcom/client/public/tla/locales-compiled/en.json

```diff
@@ -14,7 +14,6 @@
   "1da2e8106b": "An unexpected error occurred.",
   "1dd1c5fb7f": "Today",
   "1f9696d5f2": "Export the content as a .tldr file: Go to 'File' > 'Save a copy'",
-  "20530afec1": "{count, plural, one {Uploading {count} .tldr file…} other {Uploading {count} .tldr files…}}",
```

Comment:
Looks like this does not get removed from all other localizations. I forgot if we ever clean not used localizations from other languages? @mimecuvalo

Reply:
Hey, yeah, they won't get proactively removed from other languages until the translations update in general (which is a weekly process). I could maybe do this in advance but it becomes tricky and not super worth it.